smac/optimizer/smbo.py

```diff
@@ -425,29 +424,31 @@ def _component_builder(self, conf:typing.Union[Configuration, dict]) \
                   max_depth=conf.get("max_depth", self.scenario.rf_max_depth))
 
         elif conf["model"] == "GP":
-            cov_amp = 2
-            n_dims = len(types)
-
-            initial_ls = np.ones([n_dims])
-            exp_kernel = george.kernels.Matern52Kernel(initial_ls, ndim=n_dims)
-            kernel = cov_amp * exp_kernel
-
-            prior = DefaultPrior(len(kernel) + 1, rng=self.rng)
+            raise NotImplementedError()
+            cov_amp = skopt.learning.gaussian_process.kernels.ConstantKernel(2.0, constant_value_bounds=(1e-10, 2))
```

Comment:
Shouldn't we use our kernel code here?

Reply:
yes. This code was not used (see the exception above the line you commented). After discussing that we want to keep this code for @hfurkanbozkurt I updated the behavior of this function.